Localization/l10n.lua

```diff
@@ -0,0 +1,163 @@
+l10n = {}
+l10n.translations = {}
+
+l10n.itemLookup = {}
+l10n.npcNameLookup = {}
+l10n.objectNameLookup = {}
+l10n.objectLookup = {}
+l10n.questLookup = {}
+l10n.continentLookup = {}
+l10n.zoneLookup = {}
+l10n.zoneCategoryLookup = {}
+l10n.questCategoryLookup = {}
+
+---@type QuestieDB
+local QuestieDB = QuestieLoader:ImportModule("QuestieDB")
+
+local locale = 'enUS'
+local supportedLocals = {
+    ['enUS'] = true,
+    ['esES'] = true,
+    ['ptBR'] = true,
+    ['frFR'] = true,
+    ['deDE'] = true,
+    ['ruRU'] = true,
+    ['zhCN'] = true,
+    ['zhTW'] = true,
+    ['koKR'] = true,
+}
+
+l10n.questCategoryKeys = {
```

Comment:
This does not belong to `l10` but is only used in the Journey. Move it somewhere else